debian/control

```diff
@@ -1,19 +1,24 @@
-Source: python-tuf
+Source: tuf
 Maintainer: Debian Python Team <team+python@tracker.debian.org>
 Uploaders: Philippe Coval <rzr@users.sf.net>
-Section: devel
+Section: python
 Priority: optional
 Build-Depends: dh-python,
-               debhelper-compat (= 12),
+               debhelper-compat (= 13),
                python3-all,
-               python3-setuptools,
-               python3-six,
+               python3-coverage,
+               python3-cryptography,
+               python3-dateutil,
+               python3-nacl,
                python3-requests,
-               python3-securesystemslib
-Standards-Version: 4.5.0
-Vcs-Browser: https://github.com/theupdateframework/tuf
-Vcs-Git: https://github.com/theupdateframework/tuf.git
+               python3-securesystemslib,
+               tox
+Standards-Version: 4.5.1
+Rules-Requires-Root: no
+Vcs-Browser: https://salsa.debian.org/python-team/packages/tuf/-/tree/debian/master
```

Comment:
I recommend use `https://salsa.debian.org/python-team/packages/tuf` because here vcs-browser is pointing to a particular branch, and if that change in the future, that link would be incorrect.